### PR TITLE
Int 831

### PR DESCRIPTION
**Pull request by rmarinleal**

### CODE REVIEW

File: views/templates/hook/product-simulator.tpl [hunk 1]
```diff
@@ -37,104 +58,108 @@
...
-                    quantitySelector = findQuantitySelector();
-                    if (quantitySelector === 'default') {
-                        quantity = '1'
-                    }
+                    quantity = '1'
                 }
+            }
 
-                sdk.product_simulator = {};
-                sdk.product_simulator.id = 'product-simulator';
```
Comment (pgarcess) [inline views/templates/hook/product-simulator.tpl:87]: The id must be set buy the own simulator

File: views/templates/hook/product-simulator.tpl [hunk 2]
```diff
@@ -37,104 +58,108 @@
             window.PSSimulatorAttempts = window.attempts + 1;
             if (window.attempts > 4 )
             {
+                console.log("1");
```
Comment (pgarcess) [inline views/templates/hook/product-simulator.tpl]: remove console

File: views/templates/hook/product-simulator.tpl [hunk 3]
```diff
@@ -18,16 +18,37 @@
...
     <script>
-        function checkSimulatorContent() {
+        function checkSimulatorContent(clearDiv = false) {
             var pgContainer = document.getElementsByClassName("pagantisSimulator");
             if(pgContainer.length > 0) {
                 var pgElement = pgContainer[0];
-                if (pgElement.innerHTML != '')
+                if (pgElement.innerHTML != '' && pgElement.innerText != "{$pagantisSimPreposition|escape:'quotes'}")
                 {
+                    console.log("1content:", pgElement.innerHTML)
```
Comment (pgarcess) [inline views/templates/hook/product-simulator.tpl]: more console

File: views/templates/hook/product-simulator.tpl (hunk 1 above)
Comment (rmarinleal) [reply views/templates/hook/product-simulator.tpl:87]: you can set it if you want to know it in advance, but in this case I don use it so this line is useless

File: views/templates/hook/product-simulator.tpl (hunk 2 above)
Comment (rmarinleal) [reply views/templates/hook/product-simulator.tpl]: ops :(

File: views/templates/hook/product-simulator.tpl (hunk 3 above)
Comment (rmarinleal) [reply views/templates/hook/product-simulator.tpl]: ops v2 ... XD

File: pagantis.php [hunk 4]
```diff
@@ -409,12 +425,12 @@ public function hookPaymentOptions()
             'ps_version'                         => str_replace('.', '-', Tools::substr(_PS_VERSION_, 0, 3)),
         ));
 
-        $logo = ($this->language == 'ES' || $this->language == null) ? 'logo_pagamastarde.png' : 'logo_pagantis.png';
+        $logo = 'https://cdn.digitalorigin.com/assets/master/logos/pg-favicon.png';
         $paymentOption = new PrestaShop\PrestaShop\Core\Payment\PaymentOption();
         $paymentOption
             ->setCallToActionText($pagantisTitle)
             ->setAction($link->getModuleLink('pagantis', 'payment'))
-            ->setLogo($this->getPathUri(). 'views/img/' . $logo)
```
Comment (pgarcess) [inline pagantis.php:417]: local logo?

Comment (rmarinleal) [reply pagantis.php:417]: yes is mandatory :(

**Review by pgarcess — APPROVED** [review]

**Pull request merged**